exampleApp/src/main/java/it/trade/android/exampleapp/MainActivity.java

```diff
@@ -68,8 +68,8 @@
         DELETE_ALL_LINKED_BROKERS("Delete all linked brokers"),
         AUTHENTICATE_FIRST_LINKED_BROKER("Authenticate first linked broker"),
         AUTHENTICATE_ALL_LINKED_BROKERS("Authenticate all linked brokers"),
-        AUTHENTICATE_WITH_SECURITY_QUESTION_SIMPLE("Simple security question"),
-        AUTHENTICATE_WITH_SECURITY_QUESTION_OPTIONS("Security question with options"),
+//        AUTHENTICATE_WITH_SECURITY_QUESTION_SIMPLE("Simple security question"),
```

Comment:
Why are we removing these?

Reply:
originally it's not implemented, not used at all. in fact, to test security questions, we just need to use same `testOAuthFlow` and use `dummySecurity` and `dummyOption` instead.

Reply:
Ok, can we remove them if we are not using instead of commenting out?